core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -1653,7 +1663,19 @@ private[spark] object JsonProtocolSuite extends Assertions {
       |      "Remote Bytes Read": 1000,
       |      "Remote Bytes Read To Disk": 400,
       |      "Local Bytes Read": 1100,
-      |      "Total Records Read": 10
+      |      "Total Records Read": 10,
+      |      "Remote Requests Duration": 900,
+      |      "Push Based": {
+      |         "Corrupt Merged Block Chunks" : 100,
+      |         "Fallback Count" : 100,
```

Comment:
Curious why these are nonzero if push based shuffle was disabled ?
Are these simply values to test serde ?